apps/dashboard_reports/templates/dashboard_reports/base.html

```
@@ -0,0 +1,19 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)